src/app/UserProfilePage/UserProfilePage.jsx

```diff
@@ -1,21 +1,42 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
 import Container from '../../components/Container/Container';
 import ProfileNav from './components/ProfileNav';
 import ProfileContent from './components/ProfileContent';
 import Title from '../../components/Title';
+import getUser from '../../apis/auth/getUser';
+import LoadingState from '../EmailVerifyPage/components/LoadingState';
 
 const UserProfileContainer = styled(Container)`
   display: flex;
 `;
 
-const UserProfilePage = () => (
-  <>
-    <Title>账户信息｜用户中心</Title>
-    <UserProfileContainer>
-      <ProfileNav />
-      <ProfileContent />
-    </UserProfileContainer>
-  </>
-);
+const UserProfilePage = () => {
+  const [userProfile, setUserProfile] = useState('');
```

Comment:
这里直接叫 user 好一些 get-user 返回 user
```suggestion
  const [user, setUser] = useState();
```

Reply:
改好了！